<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab { height: 60px;}
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>TEST</string>
     </attribute>
     <widget class="QWidget" name="plot_widget" native="true">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>500</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>380</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current test No</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_current_test_no">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>380</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>420</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Status</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test_status">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>420</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_communication">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No Communication</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_main_ch4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>284</y>
        <width>240</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{  margin-top: 0px;}
</string>
      </property>
      <property name="title">
       <string>CHANNEL 4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QLabel" name="label_main_ch4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_ch4_unit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tare_ch4">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>2</y>
         <width>60</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_main_load">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>240</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{  margin-top: 0px;}
</string>
      </property>
      <property name="title">
       <string>LOAD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_main_load">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_load_unit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tare_load">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>2</y>
         <width>60</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_main_ch2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>148</y>
        <width>240</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{  margin-top: 0px;}
</string>
      </property>
      <property name="title">
       <string>CHANNEL 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QLabel" name="label_main_ch2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_ch2_unit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tare_ch2">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>2</y>
         <width>60</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_main_ch3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>216</y>
        <width>240</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{  margin-top: 0px;}
</string>
      </property>
      <property name="title">
       <string>CHANNEL 3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QLabel" name="label_main_ch3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_ch3_unit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tare_ch3">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>2</y>
         <width>60</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_displacement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>240</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{  margin-top: 0px;}
</string>
      </property>
      <property name="title">
       <string>DISPLACEMENT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QLabel" name="label_main_displacement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>--------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tare_disp">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>2</y>
         <width>60</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TARE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_main_displacement_unit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_area">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_stress">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>STRESS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PARAMETERS</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_read_parameter">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Read
Parameter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>SETTINGS</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_41">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>345</width>
        <height>95</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>THEME</string>
      </property>
      <widget class="QComboBox" name="comboBox_theme">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>35</y>
         <width>270</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionDeneme">
   <property name="text">
    <string>deneme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
